index.html

```diff
@@ -3,51 +3,50 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>myproject</title>
+    <title>Document</title>
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
-    <header>
-        <nav >
-            <div class="menu-container">
-                <ul>
-                    <li><a href="#">My logo</a></li>
-                    <li style="place-items: left;"><a href="#",></a>
-                        <div class="bar"></div>
-                        <div class="bar"></div>
-                        <div class="bar"></div>
-                    </li>
-                </ul>
+    <div class="body">
+        <div class="content">
+            <img src="c:\Users\kalyst\Downloads\Microverse-Student-Project-1 (1)\Bars\Status Bars\Dark.png">
+
+        </div>
+        <div class="head">
 
-            </div>
             
-        </nav>
-    </header>
-    <div style="background-color: #1C1A19; width: Hug (375px)px;height: Hug (590px)px; top: 92px;
-    padding: 120px 24px 120px 24px; gap: 12px;opacity: 0px;">
-        <head>
-            <div>HEY THERE</div>
-            <div>I AM ALEX</div>
-        </head>
-        <div>
-            <h1>
-                <div>I am a software developer</div>
-                <div>
-                    I can help you build a product, feature<br>
-                    or website look through some of my<br>
-                    work and experience! if you like what you<br>
-                    see and have a project what you need coded<br>
-                    dont hesitate to comment me<br>
-                </div>
-                <div>
-                    <img src="/home/codecraft/Downloads/Microverse-Student-Project-1/github.svg", alt="github">
-                    <img src="/home/codecraft/Downloads/Microverse-Student-Project-1/Linkdin icon.svg", alt="Linkdin">
```

Comment:
Please add icons and images you used to a folder in your project, for good organization of your project. those images and icons for your project should also be present in the project repository in the git-hub.